likecoin/admin/editor.php

```diff
@@ -76,6 +78,7 @@ function likecoin_add_posts_columns( $columns ) {
  * @param int|    $post_id The current post ID.
  */
 function likecoin_populate_posts_columns( $column, $post_id ) {
+	global $post;
```

Comment:
Where is `$post` used?

Reply:
here: $publish_params = likecoin_get_meta_box_publish_params( $post );